README.md

```diff
@@ -157,12 +156,12 @@ Add the following step to your GitHub workflow (in example are used non-default
     GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}  
   with:
     tag-name: "v0.1.0"
-    chapters: '[
-      {"title": "Breaking Changes 💥", "label": "breaking-change"},
-      {"title": "New Features 🎉", "label": "enhancement"},
-      {"title": "New Features 🎉", "label": "feature"},
-      {"title": "Bugfixes 🛠", "label": "bug"}
-    ]'
+    chapters:
+      - {"title": "Breaking Changes 💥", "label": "breaking-change"},
+      - {"title": "New Features 🎉", "label": "enhancement"},
+      - {"title": "New Features 🎉", "label": "feature"},
+      - {"title": "Bugfixes 🛠", "label": "bug"}
```

Comment:
See the comment:

Reply:
This format is called YAML Flow style.

Reply:
This example should not have a trailing commas.